src/schemas.py

```
@@ -420,4 +432,21 @@ class APIUsageStats(BaseModel):
+
+class APILogBase(BaseModel):
+    image_url: Optional[str] = None
+    ocr_output: Optional[str] = None
+    prompt_version: Optional[str] = None
+    user_id: Optional[str] = None
+    log_metadata: Optional[Dict[str, Any]] = None
+
+class APILogCreate(APILogBase):
+    pass
+
+class APILog(APILogBase):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)